packages/x-charts/src/ChartContainer/defaultPlugins.ts

```diff
@@ -2,5 +2,12 @@ import { plugin as barPlugin } from '../BarChart/plugin';
 import { plugin as scatterPlugin } from '../ScatterChart/plugin';
 import { plugin as linePlugin } from '../LineChart/plugin';
 import { plugin as piePlugin } from '../PieChart/plugin';
+import { ChartSeriesType } from '../models/seriesType/config';
+import { ChartsPluginType } from '../models';
 
-export const defaultPlugins = [barPlugin, scatterPlugin, linePlugin, piePlugin];
+export const defaultPlugins: ChartsPluginType<ChartSeriesType>[] = [
```

Comment:
Not wrong but less precise

```suggestion
export const defaultPlugins: ChartsPluginType<'bar' | 'scatter' | 'line' | 'pie'>[] = [
```